### PR TITLE
Reducing the number of screen + disabling undo button when can't undo more

### PR DESCRIPTION
**Pull request by ayyysh04**
Screenshot:
Screenshot_2021-11-10-03-49-18-553_com example doclense

### CODE REVIEW

File: lib/image_view.dart [hunk 1]
```diff
@@ -107,137 +148,176 @@ class _ImageviewState extends State<Imageview> {
...
-  void goToFilterImageScreen({File file}) {
-    Navigator.of(context).pushNamed(
-      RouteConstants.filterImageScreen,
-      arguments: {
-        'file': file,
-        'list': widget.list,
-      },
-    );
-  }
+  // void goToFilterImageScreen({File file, @required BuildContext context}) {
```
Comment (anushbhatia) [inline lib/image_view.dart]: remove these lines. I don't think they are needed anymore.

**Review by anushbhatia — CHANGES_REQUESTED** [review]
Great Work

**Comment by anushbhatia** [comment]
you can also resolve the linting issues by removing unused imports.

File: lib/image_view.dart (hunk 1 above)
Comment (ayyysh04) [reply lib/image_view.dart]: Done the changes. You can review it now

**Comment by ayyysh04** [comment]
@anushbhatia Done the changes,please review the same

**Comment by anushbhatia** [comment]
Great Work😇

**Pull request merged**